docs/logging-install.md

```diff
@@ -0,0 +1,148 @@
+**This document discusses setting up of logging infrastructure in the OpenShift
```

Comment:
should we think about adding a heading to document at the top?

Reply:
We're describing the purpose of the doc at the beginning. So adding a heading doesn't make sense to me.